core/build_playerglobal/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
How critical it is being a separate crate? For simplicity I prefer `build_playerglobal` would be just sit in `core` (with sacrificing the ability to be invoked manually). It's that I'm not so sure what will be a valid use-case where you really need to build `playerglobal.swf` alone.

Reply:
I've found having a standalone tool to be very useful when debugging the SWF generation - for example, when changing the command line parameters passed to `asc.jar`, or when adjusting the stubs being generated. I think this is particularly useful now that playerglobal.swf no longer has a stable path (e.g. it's something like `./target/debug/build/ruffle_core-f4e48ff77a789324/out/playerglobal.swf`)